INSTALL.md

```diff
@@ -38,6 +38,9 @@ conda activate optimade
 # Install package and dependencies in editable mode (including "dev" requirements).
 pip install -e ".[dev]"
 
+# Retrieve a list of providers. (Without this list some of the test will fail because providers.json cannot be found.)
```

Comment:
:tada: 

Think this is enough to close #370.

Very minor suggestion below:

```suggestion
# Optional: Retrieve the list of OPTIMADE providers. (Without this submodule, some of the tests will fail because "providers.json" cannot be found.)
```

Reply:
We may also want to tweak the way the tests for the providers work, now that we have lots of fallbacks.

Reply:
I am not sure what you meant with this comment, so if you want me to do something, you should clarify what I shoudl do.

Reply:
Think this is okay for now, I just noticed that the test failure that triggered this change for you might now be out of date.